edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -525,6 +525,12 @@ where
     async fn system_info(&self) -> anyhow::Result<CoreSystemInfo> {
         log::info!("Querying system info...");
 
+        let total_memory = {
+            let mut system_resources = self.system_resources.as_ref().lock().await;
+            system_resources.refresh_memory();
+            system_resources.total_memory() * 1000
```

Comment:
The implementation uses the "MemTotal" line from `/proc/meminfo`, which is in KiB (even though the file confusingly calls it "kB"), so this should be `* 1024`

Reply:
Very confusing, will fix.

Reply:
cb209a831c282bc4c42f72b15c459f931b9638f7

Reply:
Fix for two other memory query sites: af40269b2fcf231f3c0553f1d68e7db7cb8b891d